examples/latex.jl

```diff
@@ -15,13 +14,14 @@ function main(window)
              textinput() >>> texᵗ,
              hbox("Show as a block", hskip(1em), checkbox(mode) >>> modeᵗ),
              vskip(1em),
-             tex(t, block=mode)) |> pad(1em) |> maxwidth(30em)
+             tex(t, block=mode)) |> Escher.pad(1em) |> maxwidth(30em)
```

Comment:
This seems more like a philosophical issue. I got a pad warning when running the example, probably because I ran an example with Compose before it. 

So given that a multi-page site might trigger this warning where a single page doesn't, should we qualify every pad as Escher.pad? Or use the examples as the minimum necessary example (to your point, showing its not always needed)?

> On April 12, 2016, at 5:27 PM, Rohit Varkey Thankachan <notifications@github.com> wrote:
> 
> In examples/latex.jl:
> 
> > @@ -15,13 +14,14 @@ function main(window)
> >               textinput() >>> texᵗ,
> >               hbox("Show as a block", hskip(1em), checkbox(mode) >>> modeᵗ),
> >               vskip(1em),
> > -             tex(t, block=mode)) |> pad(1em) |> maxwidth(30em)
> > +             tex(t, block=mode)) |> Escher.pad(1em) |> maxwidth(30em)
> You actually don't need to qualify pad here, as we don't have Compose loaded.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>